x-pack/plugins/observability_solution/synthetics/server/synthetics_service/synthetics_service.ts

```diff
@@ -307,9 +307,10 @@ export class SyntheticsService {
     return this.server.coreStart?.elasticsearch.client.asInternalUser;
   }
 
-  async getOutput() {
+  async getOutput({ inspect }: { inspect: boolean } = { inspect: false }) {
     const { apiKey, isValid } = await getAPIKeyForSyntheticsService({ server: this.server });
-    if (!isValid) {
+    // do not check for api key validity if inspecting
+    if (!isValid && !inspect) {
```

Comment:
I noticed when I was adjusting the inspect tests, that they would sometime fail on api key not valid. No need to check api key when doing an inspect.